draft-the-rockstar-killer-ninja-10xcoder-and-you.markdown

```diff
@@ -0,0 +1,105 @@
+
+I have been doing tech recruiting for start-ups over the last 10 years and oh boy, did I fuck up a lot. On the plus side of things, I did also learn a lot about human behaviour, [hiring bias](http://recruiterbox.com/blog/remove-hiring-bias-from-recruitment-process/) and common misconceptions about what constitutes a "good" team member. 
+
+I have seen the biggest douche bags called the "top performers" and the actual performers "constant underachievers". Actually that's a pretty common topic I have seen across companies. Human perception is pretty freakin' bad. We overvalue merits that are tangible and undervalue merits that are kind of fuzzy. For instance, a classical set up I probably have seen dozens of time is the rockstar developer (sometimes also referred to as [ninja](TODO) or [killer](TODO))that delivers this super useful new backend feature that will be used on the web frontend everybody has been waiting for. 
+
+Problem is that he didn't really care about communicating this. QA is missing. Normally not an issue since developers work in a squad where QA is a part of the team (you're doing this, right?) but that rockstar never actually fit in anywhere in so you kind of let him work on his own (what could possibly go wrong?). Supervision by the product team was spotty at best since nobody there really got along with our that dev (which imposes a high entry barrier for communications) and as a consequence the killer feature everybody has been waiting is not **really** the thing that people have been expecting. Customer Care doesn't know about it. And the frontend team has no clue that they should integrate this feature and more importantly, how.
+6 months later - the feature still not being live - your CEO barges in and asks you why the hell it is still not live.
+
+Now there are obviously a lot of things that went wrong (and not just with that single developer) but in most cases the overall perception will be that the rockstar developer delivered. While the other teams failed to so.
+
+**So how can you make sure this doesn't happen to you?**
+
+Apart from the organisational problems the obvious solution to this is not hiring those kind of guys. The problem is that "common recruiting wisdom" says to look for developers that are technically capable first and then look for their so called "soft skills" (which is a very fuzzy term which roughly translates to "the candidate should not be an obvious asshole").
+
+I think the way most companies evaluate candidates for tech jobs (probably for all jobs) is upside down.
+
```

Comment:
Header:

Conventional recruiting wisdom is outdated and broken

.....
.....

Now the question becomes: What should youactually look for in a candidate?